src/main/java/uk/gov/companieshouse/overseasentitiesapi/converter/Version_2_0_DocumentTransformer.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.companieshouse.overseasentitiesapi.converter;
+
+import org.bson.Document;
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.overseasentitiesapi.model.SchemaVersion;
+import uk.gov.companieshouse.overseasentitiesapi.utils.ApiLogger;
+
```

Comment:
I'd add a class level comment (similar to the one in version 1.0) to outline what needs to happen in terms of moving the name field for a version 2.0